### PR TITLE
Added several programming languages and fixed typos

### PR DESCRIPTION
**Pull request by rafa-diniz**
Updated the programming languages list by using a json I found on https://gist.github.com/aymen-mouelhi/82c93fbcd25f091f2c13faa5e0d61760
All credit to https://gist.github.com/aymen-mouelhi for creating the json.

Fixed a typo (missing quotation mark) on "Media/Adobe/Photoshop".

PS: Sorry for comming 2 times the same thing, I'm quite new to using github :)

### CODE REVIEW

**Comment by sreekaransrinath** [comment]
Damn son, that's some good work you've done there.

Before I pull it in though, have you tested the script to see if the directories are built correctly? Because I can see some potential points of pressure in this.

**Comment by rafa-diniz** [comment]
Yeah, turns out the backwards slash wasn't necessary on the directory name. 
I'll also remove some single quotes used in the directories' names, because it creates a dir with double quotes, and it makes it more time-consuming to cd to it.

I aside from that, everything seems to be working fine. I'll generate another commit with the file fixed :P

File: extensiondict.py [hunk 1]
```diff
@@ -177,5 +161,736 @@ def getpath(extension):
...
+    '.lisp'                   : "Programming/Common\ Lisp",
+    '.asd'                    : "Programming/Common\ Lisp",
+    '.cl'                     : "Programming/Common\ Lisp",
+    '.l'                      : "Programming/Common\ Lisp",
+    '.lsp'                    : "Programming/Common\ Lisp",
+    '.ny'                     : "Programming/Common\ Lisp",
+    '.podsl'                  : "Programming/Common\ Lisp",
+    '.sexp'                   : "Programming/Common\ Lisp",
+    '.cp'                     : "Programming/Component\ Pascal",
+    '.cps'                    : "Programming/Component\ Pascal",
```
Comment (sreekaransrinath) [inline extensiondict.py]: Remove backslashes here

File: extensiondict.py [hunk 2]
```diff
@@ -177,5 +161,736 @@ def getpath(extension):
...
+    '.chs'                    : "Programming/C2hs\ Haskell",
+    '.clp'                    : "Programming/CLIPS",
+    '.cmake'                  : "Programming/CMake",
+    '.cmake.in'               : "Programming/CMake",
+    '.cob'                    : "Programming/COBOL",
+    '.cbl'                    : "Programming/COBOL",
+    '.ccp'                    : "Programming/COBOL",
+    '.cobol'                  : "Programming/COBOL",
+    '.cpy'                    : "Programming/COBOL",
+    '.capnp'                  : "Programming/Cap'n\ Proto",
```
Comment (sreekaransrinath) [inline extensiondict.py]: Remove backslash and single quote

File: extensiondict.py [hunk 3]
```diff
@@ -177,5 +161,736 @@ def getpath(extension):
...
+    '.h++'                    : "Programming/C++",
+    '.hh'                     : "Programming/C++",
+    '.hpp'                    : "Programming/C++",
+    '.hxx'                    : "Programming/C++",
+    '.inc'                    : "Programming/C++",
+    '.inl'                    : "Programming/C++",
+    '.ipp'                    : "Programming/C++",
+    '.tcc'                    : "Programming/C++",
+    '.tpp'                    : "Programming/C++",
+    '.chs'                    : "Programming/C2hs\ Haskell",
```
Comment (sreekaransrinath) [inline extensiondict.py]: Remove backslash

File: extensiondict.py [hunk 4]
```diff
@@ -177,5 +161,736 @@ def getpath(extension):
     '.lnk'   : "Text/Other/System",
     '.msi'   : "Text/Other/System",
     '.sys'   : "Text/Other/System",
-    '.tmp'   : "Text/Other/System"
-}
+    '.tmp'   : "Text/Other/System",
+    # Programming
+    '.abap'                   : "Programming/ABAP",
+    '.asc'                    : "Programming/AGS\ Script",
+    '.ash'                    : "Programming/AGS\ Script",
```
Comment (sreekaransrinath) [inline extensiondict.py]: Remove backslash

File: extensiondict.py [hunk 5]
```diff
@@ -177,5 +161,736 @@ def getpath(extension):
...
+    '.feature'                : "Programming/Cucumber",
+    '.cu'                     : "Programming/Cuda",
+    '.cuh'                    : "Programming/Cuda",
+    '.cy'                     : "Programming/Cycript",
+    '.pyx'                    : "Programming/Cython",
+    '.pxd'                    : "Programming/Cython",
+    '.pxi'                    : "Programming/Cython",
+    '.d'                      : "Programming/D",
+    '.di'                     : "Programming/D",
+    '.com'                    : "Programming/DIGITAL\ Command\ Language",
```
Comment (sreekaransrinath) [inline extensiondict.py]: Remove backslash

File: extensiondict.py [hunk 6]
```diff
@@ -177,5 +161,736 @@ def getpath(extension):
...
+    '.ecl'                    : "Programming/ECL",
+    '.eclxml'                 : "Programming/ECL",
+    '.ecl'                    : "Programming/ECLiPSe",
+    '.e'                      : "Programming/Eiffel",
+    '.ex'                     : "Programming/Elixir",
+    '.exs'                    : "Programming/Elixir",
+    '.elm'                    : "Programming/Elm",
+    '.el'                     : "Programming/Emacs\ Lisp",
+    '.emacs'                  : "Programming/Emacs\ Lisp",
+    '.emacs.desktop'          : "Programming/Emacs\ Lisp",
```
Comment (sreekaransrinath) [inline extensiondict.py]: Remove backslash

**Review by sreekaransrinath — APPROVED** [review]
Alright @RADA77777 , looks good!

**Pull request merged**